tests/slow/todo.diff

```diff
@@ -0,0 +1,208 @@
+--- /home/gui/scala-syntax/tests/slow/target/repos/scalafx/scalafx/src/test/scala/scalafx/util/DurationSpec.scala
```

Comment:
This is the remaining 15 files from `slow/test` maybe I should not commit this file.